pandas/core/generic.py

```diff
@@ -5718,30 +5718,34 @@ def clip_lower(self, threshold, axis=None, inplace=False):
         """
         Return copy of the input with values below given value(s) trimmed.
 
-        If an element value is below the threshold, the threshold value is
-        returned instead.
+        Elements below the `threshold` will be changed to `threshold`.
 
         Parameters
         ----------
-        threshold : float or array_like
+        threshold : float or array-like
             Lower value(s) to which the input value(s) will be trimmed.
-        axis : {0 or 'index', 1 or 'columns'}
+        axis : {0 or ‘index’, 1 or ‘columns’, None}, default None
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0.
+            Whether to perform the operation in place on the data.
+
+            .. versionadded:: 0.21.0
 
         See Also
         --------
-        DataFrame.clip: Trim values at input threshold(s).
-        DataFrame.clip_upper: Return copy of input with values above given
+        DataFrame.clip : Trim values at input threshold(s).
```

Comment:
Some suggested better wording:

`Series.clip` : General purpose method to trim `DataFrame` values to given threshold(s)
`DataFrame.clip` : General purpose method to trim `Series` values to given threshold(s)
`Series.clip_lower` : Trim `Series` values below given threshold(s)
`Series.clip_upper` : Trim `Series` values above given threshold(s)
`DataFrame.clip_upper` : Trim `DataFrame` values above given threshold(s)

Reply:
Done, but changed the order to "more related" to "less related".